linux_os/guide/services/fapolicyd/service_fapolicyd_enabled/rule.yml

```diff
@@ -19,6 +19,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000480-GPOS-00227
```

Comment:
@tedbrunell should we add `SRG-OS-000370-GPOS-00155` to this rule as well?

Reply:
SRG-OS-000480-GPOS-00227 |   | The operating system must be configured in accordance with the security configuration settings based on DoD security configuration or implementation guidance, including STIGs, NSA configuration guides, CTOs, and DTMs.
-- | -- | --


vs


SRG-OS-000370-GPOS-00155 |   | The operating system must employ a deny-all, permit-by-exception policy to allow the execution of authorized software programs.
-- | -- | --


Seems like a good suggestion. Quick questions:
1) Does fapolicyd default to deny-all, to align with the SRG-OS-000370-GPOS-00155
2) Is there an equivalent of audit.rules that should be checked, to ensure fapolicyd is set to deny all?

Reply:
Fapolicyd is deny all, permit-by-exception where the exception is whether or not the application is known in the rpm database.

Reply:
swapped to SRG-OS-000370-GPOS-00155